src/overlays/actors/ovl_En_Ruppecrow/z_en_ruppecrow.h

```diff
@@ -3,15 +3,51 @@
 
 #include "global.h"
 
+#define EN_RUPPECROW_RUPEE_COUNT (20)
+#define EN_RUPPECROW_LIMB_COUNT (9)
+#define EN_RUPPECROW_LIMB_POS_COUNT (4)
+
+#define EN_RUPPECROW_SHOULD_CRY(globalCtx) ((globalCtx->state.frames % 43) == 0)
+
+#define EN_RUPPECROW_OSCILLATE_Y_OFFSET(this, addition, multiplier) do { \
+        this->yOffset += addition; \
+        this->actor.shape.yOffset = Math_SinS(this->yOffset) * multiplier; \
+    } while (false);
```

Comment:
Is this macro required? We prefer not to use compound macros like this unless the code requires it.